src/insights_client/__init__.py

```diff
@@ -398,13 +421,17 @@ class InsightsConstants(object):
 
         # handle client instantation here so that it isn't done multiple times in __init__
         # The config can be passed now by parameter
+        logger.debug("Initializing InsightsClient with the loaded config...")
         client = InsightsClient(config, False)  # read config, but dont setup logging
+        logger.debug("InsightsClient initialized. Egg version: %s" % (client.version(),))
```

Comment:
From these I would only log the second one.

```py
logger.debug("Client initialized with egg '%s'.", client.version())
```

Reply:
Do you think the first message is not useful? _InsightsClient_ initialization is a big thing and in case of failure, it may be nice to see the first message as the pointer to the last place where things we ok.